Testing/Temporary/LastTest.log

```diff
@@ -0,0 +1,3 @@
+Start testing: Jul 14 10:51 CDT
+----------------------------------------------------------
+End testing: Jul 14 10:51 CDT
```

Comment:
Same as with `Testing/Temporary/CTestCostData.txt`.

I'd dare to say that the whole `Temporary` folder should not be committed.

Reply:
Removed and .gitignore added.